### PR TITLE
refactor(dedicated): remove user service from billing

### PR DESCRIPTION
**Pull request by frenautvh**
| Question         | Answer
| ---------------- | ---
| Branch?          | develop
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | internal MANAGER-7110
| License          | BSD 3-Clause

## Description

remove User service dependency from billing module

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/account/contacts/request/user-contacts-request.controller.js [hunk 1]
```diff
@@ -54,7 +41,7 @@ export default /* @ngInject */ function UserAccountContactsRequestsController(
     self.loaders.init = true;
     self.addMode = false;
     self.contactTasksDetails = [];
-    $q.all([getUser(), getTaskStateEnum()])
+    $q.all([getTaskStateEnum()])
```
Comment (radireddy) [inline packages/manager/apps/dedicated/client/app/account/contacts/request/user-contacts-request.controller.js:44]: may be you can remove $q.all?

File: packages/manager/apps/dedicated/client/app/billing/confirmTerminate/legacy/termination-legacy.controller.js [hunk 2]
```diff
@@ -48,21 +47,14 @@ export default class TerminateServiceCtrl {
       return;
     }
 
-    this.$q.all([this.loadUser(), this.loadService()]).catch(() => {
+    const { ovhSubsidiary } = this.coreConfig.getUser();
+    this.USVersion = ovhSubsidiary === 'US';
```
Comment (marie-j) [inline packages/manager/apps/dedicated/client/app/billing/confirmTerminate/legacy/termination-legacy.controller.js:51]: Next step: Feature flipping

File: packages/manager/apps/dedicated/client/app/billing/confirmTerminate/legacy/termination-legacy.html [hunk 3]
```diff
@@ -39,7 +39,7 @@
         </oui-header>
 
         <div>
-            <div data-ng-if="!TerminateServiceCtrl.userLoading">
+            <div>
```
Comment (marie-j) [inline packages/manager/apps/dedicated/client/app/billing/confirmTerminate/legacy/termination-legacy.html:42]: Both `div` are still needed ?

**Review by Heiiji — APPROVED** [review]

**Review by marie-j — DISMISSED** [review]

**Comment by antleblanc** [comment]
@frenautvh Could you please resolve existing conflicts?
Thank you!

**Comment by frenautvh** [comment]
rebased on develop 31c0c68b29d489b92238efd370eb417d01e5f247

**Review by marie-j — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/billing/payment/method/controller.js [hunk 4]
```diff
@@ -21,6 +23,9 @@ export default class BillingPaymentMethodCtrl {
     this.tableFilterOptions = null;
     this.guide = null;
```
Comment (cbourgois) [inline packages/manager/apps/dedicated/client/app/billing/payment/method/controller.js:24]: NP : we can remove this one ;)

**Review by cbourgois — APPROVED** [review]

**Pull request merged**